app/components/SelectSpecies/index.js

```diff
@@ -333,16 +340,21 @@ const SelectSpecies = ({
 
   const onPressTreeCountNextBtn = () => {
     let speciesListClone = speciesList;
-    let specie = speciesListClone[activeSpeice];
+    console.log(typeof speciesList, 'speciesList Type')
+    let specie = speciesListClone[index];
     specie.treeCount = Number(treeCount) ? treeCount : undefined;
-    speciesListClone.splice(activeSpeice, 1, specie);
+    console.log(speciesListClone,specie.treeCount,'specie.treeCount', index, specie.scientificName);
+    speciesListClone.splice(index, 1, specie);
```

Comment:
App crashes for me when trying to selected a species for my registered trees:
![Simulator Screen Shot - iPhone 11 - 2021-01-28 at 10 46 20](https://user-images.githubusercontent.com/1532418/106120038-74ebf780-6188-11eb-849f-e9af8b5320e2.png)

Reply:
This happens after entering the number of planted trees having chosen to register "Multiple Trees" + "Off Site - Point":
![Simulator Screen Shot - iPhone 11 - 2021-01-28 at 10 52 37](https://user-images.githubusercontent.com/1532418/106120728-4b7f9b80-6189-11eb-8332-1da53f1f12fe.png)

Reply:
Creating a different branch to solve multiple trees issue.